src/main/kotlin/oncall/util/LocalDateUtil.kt

```diff
@@ -0,0 +1,55 @@
+package oncall.util
+
+import java.time.DayOfWeek
+import java.time.LocalDate
+import java.time.Month
+
+private const val DEFAULT_YEAR = 2023
+private const val i = 1
+
+private val publicHolidays = setOf(
+    LocalDate.of(DEFAULT_YEAR, 1, 1),
+    LocalDate.of(DEFAULT_YEAR, 3, 1),
+    LocalDate.of(DEFAULT_YEAR, 5, 5),
+    LocalDate.of(DEFAULT_YEAR, 8, 15),
+    LocalDate.of(DEFAULT_YEAR, 10, 3),
+    LocalDate.of(DEFAULT_YEAR, 10, 9),
+    LocalDate.of(DEFAULT_YEAR, 12, 25),
+)
+private val dayOfWeekToKor = mapOf(
+    DayOfWeek.MONDAY to "월",
+    DayOfWeek.TUESDAY to "화",
+    DayOfWeek.WEDNESDAY to "수",
+    DayOfWeek.THURSDAY to "목",
+    DayOfWeek.FRIDAY to "금",
+    DayOfWeek.SATURDAY to "토",
+    DayOfWeek.SUNDAY to "일",
+)
+private val KorToDayOfWeek = dayOfWeekToKor.entries
+    .associate { (k, v) -> v to k }
+
+fun LocalDate.isWorkingDay() = this.dayOfWeek.value in 1..5 && !this.isPublicHoliday()
+
+fun LocalDate.isNonWorkingDay() = this.dayOfWeek.value in 6..7 || this.isPublicHoliday()
+
+fun LocalDate.isPublicHoliday() = publicHolidays.contains(this)
+
+fun LocalDate.korDayOfWeek() = dayOfWeekToKor[this.dayOfWeek]!!
+
+fun LocalDate.previous() = this.minusDays(1)!!
+
+fun LocalDate.next() = this.plusDays(1)!!
```

Comment:
```suggestion
fun LocalDate.previous(): LocalDate = this.minusDays(1)

fun LocalDate.next(): LocalDate = this.plusDays(1)
```
you can declare the return type rather than use `!!`

Reply:
oh I just wanted to avoid a warning. thanks for suggesting the alternative. https://github.com/kotlin-study-dh/kotlin-oncall-6/pull/1/commits/67c850f5fcecef9594c597e307f3a5c9a73ac753